src/app/static/src/tests/components/projects/shareProject.test.ts

```diff
@@ -296,10 +371,10 @@ describe("ShareProject", () => {
         input.setValue("testing");
         input.trigger("blur");
 
-        await Vue.nextTick();
-
-        wrapper.find(Modal).find("button").trigger("click");
-        expect(cloneProject.mock.calls[0][1]).toEqual({projectId: 1, emails: ["testing"]});
+        setTimeout(() => {
```

Comment:
Interesting, I'm not familiar on the nuance of how `Vue.nextTick()` and `setTimeout()` differ.